Apps/Sandcastle/gallery/3D Tiles 1.1 Photogrammetry.html

```diff
@@ -0,0 +1,410 @@
+            <span id="leftBenchmarkNotice" class="benchmarkNotice"></span>
+            <br />
+            Tile Load Time (s): <span id="leftTileLoadTime">---</span>
+            <br />
+          </div>
+        </div>
+        <div id="right" class="statsPane panel">
+          <div id="rightTitle" class="statsTitle">---</div>
+          <div id="rightStats">
+            Tiles Loaded: <span id="rightTilesLoaded">---</span> /
+            <span id="rightTilesTotal">---</span>
+            <br />
+            GPU Memory: <span id="rightGpuMemoryMB">---</span> MB
+            <br />
+            <span id="rightBenchmarkNotice" class="benchmarkNotice"></span>
+            <br />
+            Tile Load Time (s): <span id="rightTileLoadTime">---</span>
+            <br />
+          </div>
+        </div>
+      </div>
+    </div>
+    <div id="loadingOverlay"><h1>Loading...</h1></div>
+    <div id="toolbar" class="panel">
+      <div id="toolbarSelect"></div>
+      <div id="maxSSE">
+        Maximum Screen Space Error
+        <input
+          type="range"
+          min="0.0"
+          max="64.0"
+          step="0.1"
+          data-bind="value: maximumScreenSpaceError, valueUpdate: 'input'"
+        />
+        <input
+          type="text"
+          size="5"
+          data-bind="value: maximumScreenSpaceError"
+        />
+      </div>
+    </div>
+    <script id="cesium_sandcastle_script">
+      window.startup = async function (Cesium) {
+        "use strict";
+        //Sandcastle_Begin
+
+        const ellipsoidProvider = new Cesium.EllipsoidTerrainProvider();
+
+        const updateTerrainFunc = {
+          "AGI HQ": (viewer) => {
+            viewer.scene.setTerrain(Cesium.Terrain.fromWorldTerrain());
+          },
+          Melbourne: (viewer) => {
+            viewer.terrainProvider = ellipsoidProvider;
+          },
+        };
+
+        // Tilesets produced by the Cesium ion 3D Model Tiler
+        const currentGenAssetIds = {
+          "AGI HQ": 40866,
+          Melbourne: 69380,
+        };
+
+        // Tilesets produced by the Cesium ion Reality Tiler
+        const nextGenAssetIds = {
+          "AGI HQ": 2325106,
+          Melbourne: 2325107,
+        };
+
+        // To make the code more reusable, use "left/right" terminology
+        const leftAssetIds = currentGenAssetIds;
+        const rightAssetIds = nextGenAssetIds;
+
+        const leftTitle = "3D Tiles 1.0 (JPEG textures)";
+        const rightTitle = "3D Tiles 1.1 (KTX2 textures)";
+
+        document.getElementById("leftTitle").innerHTML = leftTitle;
+        document.getElementById("rightTitle").innerHTML = rightTitle;
+
+        // Get the complete list of tileset names.
+        function getTilesetNames() {
+          const leftNames = Object.keys(leftAssetIds);
+          const rightNames = Object.keys(rightAssetIds);
+          const unique = new Set([...leftNames, ...rightNames]);
+          return [...unique].sort();
+        }
+
```

Comment:
While reusability is usually a good thing, Sandcastle examples serve a different purpose than product code. For these examples, IMO it's best to keep the code as simple as possible so a user can easily follow along. It's OK if things become a bit more manual and hardcoded.